### DISCUSSION TITLE
Cross-device beatmap synchronisation

### DISCUSSION THREAD

**Discussion by Gamingzone123** [ppy/osu#22770] [Q&A] [answered]
I would like to sync beatmaps between two computers, laptop and desktop, as such I am using a program to sync the "files" folder and with it all of the hashed files/beatmaps/folders within, this works to the extent that the files sync, but the game doesn't recognise them, can anyone think of a way to do this?

**Comment by peppy** [comment, marked as answer]
You'd need to also synchronise the database file (`client.realm`).

**Reply by Gamingzone123** [reply]
Cheers
